### ISSUE TITLE
Reproduce this repo，but the target accuracy is not achieved

### ISSUE THREAD

**Issue by Bailin-He** [sunsmarterjie/yolov12#121]
train as this
```
from ultralytics import YOLO

model = YOLO('yolo12n.yaml')

# Train the model
results = model.train(
  data='coco.yaml',
  epochs=600, 
  batch=256, 
  imgsz=640,
  scale=0.5,  # S:0.9; M:0.9; L:0.9; X:0.9
  mosaic=1.0,
  mixup=0.0,  # S:0.05; M:0.15; L:0.15; X:0.2
  copy_paste=0.1,  # S:0.15; M:0.4; L:0.5; X:0.6
  device="0,1,2,3",
  save_dir='runs/train',
  name='yolov12n_Reproduce'
)
```

and the results 

![Image](https://github.com/user-attachments/assets/f82c7418-790d-4879-9bb7-0d3684881808)

![Image](https://github.com/user-attachments/assets/0b033fae-8811-4f79-9bd8-dbb38600d9b4)

Do I need to change any configuration？

**Comment by sunsmarterjie** [comment]
Hi, did you install flash attention successfully?

**Comment by Bailin-He** [comment]
Whether this is the cause
![Image](https://github.com/user-attachments/assets/952c8674-da90-468f-8f92-7dc4efdf77df)

**Comment by sunsmarterjie** [comment]
No. Could you report the env and GPU details?
